test/MinCostFlow/test_mcf.py

```diff
@@ -0,0 +1,62 @@
+import pytest
+import shutil
+from pathlib import Path
+from src.mincostflow import MinCostFlow
+from test.MEO.test_meo import OUT_FILE
+
+TEST_DIR = 'test/MinCostFlow/'
+OUT_FILE = TEST_DIR+'output/mincostflow-output.txt'
+
+class TestMinCostFlow:
+    def test_mincostflow_required(self):
+        out_path = Path(OUT_FILE)
+        out_path.unlink(missing_ok=True)
+
+        MinCostFlow.run(sources=TEST_DIR + 'input/sources.txt',
+            targets=TEST_DIR + 'input/targets.txt',
+            edges=TEST_DIR + 'input/edges.txt',
+            output_file=OUT_FILE)
+        assert out_path.exists()
+
+    def test_mincostflow_missing_capacity(self):
```

Comment:
We should think about whether we need additional test cases to observe what happens when there is too much flow to push through the network.